src/record.js

```diff
@@ -425,16 +439,24 @@ export default class Record extends Feature {
   }
 
   get geometryAsGeoJSON() {
-    if (!this.hasCoordinate) {
-      return null;
+    if (this.isGeometryEnabled && this.geometry) {
```

Comment:
I don't think we need to check `isGeometryEnabled` here, it wasn't checking it before to return the `latitude` & `longitude`. `isGeometryEnabled` is not a new thing, it's just the "is location enabled?" setting, but I don't think we NEED to assert that here. Neither of the mobile apps have this check either for the simple accessor on a record.